app/main.py

```
@@ -361,24 +362,35 @@ async def upload_voice_with_question(
+    try:
+        message = get_frequency_result(db, username=username, is_care=False)
+        voice_service = get_voice_service(db)
+        base = voice_service.get_user_emotion_monthly_frequency(username, month)
+        frequency = base.get("frequency", {}) if base.get("success") else {}
+        return FrequencyAnalysisCombinedResponse(message=message, frequency=frequency)
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=f"분석 실패: {str(e)}")
 
-@users_router.get("/voices/analyzing/weekly")
+@users_router.get("/voices/analyzing/weekly", response_model=WeeklyAnalysisCombinedResponse)
 async def get_user_emotion_weekly(username: str, month: str, week: int, db: Session = Depends(get_db)):
```

Ruff (0.14.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)